contractsSol5/KyberMatchingEngine.sol

```diff
@@ -538,6 +539,7 @@ contract KyberMatchingEngine is KyberHintHandler, IKyberMatchingEngine, Withdraw
             }
 
             //take into account possible additional networkFee
+            require(tData.networkFeeWei + tData.platformFeeWei >= tData.networkFeeWei, "fee overflow");
```

Comment:
This check is redundant as we already had the same check right before calling this function